tfhe/src/integer/server_key/radix/mod.rs

```diff
@@ -100,7 +100,7 @@ impl ServerKey {
     /// # Example
     ///
     ///```rust
-    /// use tfhe::integer::{gen_keys_radix, IntegerCiphertext};
+    /// use tfhe::integer::{gen_keys_radix, IntegerCiphertext, IntegerCiphertextRef};
```

Comment:
I think you had a plan for an integer prelude ? could it be the time to do it ? maybe not in this PR but I think you mentioned it recently

Reply:
Yes i planned on adding the prelude after this PR as we don't want users (and potentially us) to have to remember similar but differently named traits

Reply:
fine by me